CHANGELOG.md

```diff
@@ -179,6 +179,10 @@ The `SubtreeCache.GetNodeHash()` method is no longer exported.
 The memory storage provider has been refactored to make it more consistent with
 the other storage providers.
 
+`NodeReader.GetMerkleNodes` now must return `Node` objects in the same order as
```

Comment:
This was the original intention but the documentation may have accidentally fallen into the sea ...

Reply:
Yeah, this paragraph is probably redundant. Are you suggesting to drop it?

Reply:
I think I'd leave it in as I don't think we've documented it properly before.